src/test/java/ca/intelliware/ihtsdo/mlds/web/rest/AuditResourceTest.java

```diff
@@ -100,19 +112,18 @@ public void findByFilterShouldFailForUnparseableDateRange() throws Exception {
 	        restUserMockMvc.perform(get(Routes.AUDITS)
 	        		.param("$filter", "auditEventDate ge '2014-01-02' and auditEventDate le 'NOT A DATE'")
 	                .accept(MediaType.APPLICATION_JSON_UTF8))
-	                .andExpect(status().is5xxServerError())
+	                .andExpect(status().is4xxClientError())
```

Comment:
@jaykamal just to confirm this response change was expected?

Reply:
@dmcgihtsdo after upgrading date modules as part of spring boot upgrade now we are getting 400 bad request instead of 500 so we updated the test accordingly.